cucumber/api/features/step_definitions/user_steps.rb

```diff
@@ -18,6 +18,12 @@
   end
 end
 
+Given(/^I have host "([^"]*)"$/) do |login|
```

Comment:
This also isn't used in code, I assume we will need this for later?

Reply:
Yes, i need it in the azure tests i'm writing